PennMobile/Events/Views/PennEventCellView.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  PennEventCellView.swift
+//  PennMobile
+//
+//  Created by Jacky on 3/3/24.
+//  Copyright © 2024 PennLabs. All rights reserved.
+//
+
+import SwiftUI
+
+struct PennEventCellView: View {
+    @ObservedObject var viewModel: PennEventViewModel
+    
+    let screenWidth = UIScreen.main.bounds.width
+    
+    @Environment(\.colorScheme) var colorScheme
+    
+    var shadowColor: Color {
+        colorScheme == .dark ? .white.opacity(0.2) : .black.opacity(0.2)
+    }
+    
+    var body: some View {
+        ZStack {
+            if let imageUrl = viewModel.imageUrl {
+                AsyncImage(url: imageUrl) { phase in
```

Comment:
Maybe use `KFImage` instead, that way images should be cached and don't load each time